cmake/FindLibUV.cmake

```diff
@@ -89,6 +99,16 @@ if(HAVE_LIBSENDFILE)
   list(APPEND LIBUV_LIBRARIES sendfile)
 endif()
 
+check_library_exists(userenv GetUserProfileDirectoryW "userenv.h" HAVE_LIBUSERENV)
+if(HAVE_LIBUSERENV)
+  list(APPEND LIBUV_LIBRARIES userenv)
+endif()
+
+check_library_exists(ws2_32 WSAStartup "winsock2.h" HAVE_LIBWS232)
+if(HAVE_LIBWS232)
+  list(APPEND LIBUV_LIBRARIES ws2_32)
+endif()
```

Comment:
For grouping purposes it's probably still a good idea to wrap your changes here in one big `if(MSVC) ...` block. It will help us to keep Windows-specific changes in isolated blocks as much as possible.